### DISCUSSION TITLE
Subject: Request for Local Currency Conversion Feature

### DISCUSSION THREAD

**Discussion by tonilopezortiz78** [phantom/docs#311] [General]
Subject: Request for Local Currency Conversion Feature

Dear Phantom Wallet Support Team,

I hope this message finds you well. I am writing to express my enthusiasm for Phantom Wallet and to suggest a feature that I believe would greatly enhance its functionality, particularly for users outside the United States.

Current Usage:
I currently manage my crypto assets using various wallets including Cake Wallet for Monero, Lightning for Bitcoin, and Muun for Bitcoin, alongside Phantom for Solana transactions. While Phantom excels in its core offerings, there's a feature I find vital for global usability that seems to be missing.

Issue:
Lack of Local Currency Conversion: When I'm in Thailand or any other country, I face challenges in using Phantom for transactions in local currency. For example, if I want to transfer the equivalent of 100 Thai Baht (THB) or 100 Euros to another wallet in SOL or USDT, I must first convert this amount manually using an external service, which is not user-friendly.

Feature Request:
Fiat Currency Integration: 
Local Currency Display: Enable users to view their wallet balance in their local fiat currency or any preferred currency.
Direct Fiat Conversion: Allow users to set transaction amounts in fiat currency, with the wallet automatically calculating and sending the equivalent in cryptocurrency (like SOL or USDT).

Benefits:
Enhanced Usability: This feature would make Phantom Wallet more intuitive and user-friendly for international users, eliminating the need for external conversion tools.
Global Appeal: It would cater to a broader audience, making cryptocurrency transactions as straightforward as traditional banking for those outside the US, thus promoting the adoption of cryptocurrency globally.

Implementation Example:
In wallets like Cake or Muun, users can go into settings and select their preferred fiat currency for all transactions and balance displays. This feature not only simplifies the transaction process but also makes the wallet more relatable and practical for everyday use in diverse economic environments.

I understand that implementing such features involves complexities, especially regarding real-time exchange rates and compliance with different financial jurisdictions. However, I believe this would be a significant step forward for Phantom Wallet in becoming a truly international platform.

Thank you for considering this suggestion. I look forward to your thoughts or any updates on this feature.

Best Regards,

Tony

**Comment by adamdelphantom** [comment]
Hello @tonilopezortiz78,

Please consider submitting your feature request using this [board.](https://phantom.canny.io)
